src/rev_ai/models/account.py

```diff
@@ -11,10 +11,16 @@ def __init__(self, email, balance_seconds):
         self.email = email
         self.balance_seconds = balance_seconds
 
+    def __eq__(self, other):
```

Comment:
@k-weng. I thought we discussed this be we can extend it to a more general question.That is, do we actually want to override the default equality operator? I say no, since from a user perspective, isn't this not obvious that we do so.  So say I return two `account` objects from this SDK, as a developer, I would expect them to NOT have equality as an object (In c# the default is that this implies it's referencing the same pointer in memory, I'm assuming this is the same in python). I would remove these equality overrides and just create a helper for testing shallow equality between two objects.

Reply:
meh, this isn't a language with pointer manipulation. I think this good.

If you actually wanted to know if something points to the same thing in memory, in python, you would use `id`. But nobody does that.

Reply:
@kennygea I originally had this override before our discussion and hadn't changed it yet, but now I think this is fine - `__eq__` or `==` should imply equality in attributes. In Python to check for reference equality, you would use `is`. In addition, the helper for assert would simply just be the same implementation as the overridden equality implementation.

Reply:
right, 
```
In [147]: a = set([1,2])

In [148]: b = set([2,1])

In [149]: id(a), id(b)
Out[149]: (139984634142792, 139984634143688)

In [150]: a == b
Out[150]: True

In [151]: a, b
Out[151]: ({1, 2}, {1, 2})

In [152]: a is b
Out[152]: False
```